modules/sdk-coin-avaxp/src/lib/transactionBuilder.ts

```diff
@@ -75,64 +111,10 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
     return this;
   }
 
-  /**
-   * Initialize the transaction builder fields using the decoded transaction data
-   *
-   * @param {Transaction} tx the transaction data
-   */
-  initBuilder(tx: Tx): this {
-    const baseTx = tx.getUnsignedTx().getTransaction();
-    if (
-      baseTx.getNetworkID() !== this._transaction._networkID ||
-      !baseTx.getBlockchainID().equals(this._transaction._blockchainID)
-    ) {
-      throw new Error('Network or blockchain is not equals');
-    }
-    this._transaction._memo = baseTx.getMemo();
-
-    // good assumption: addresses that unlock the outputs, will also be used to sign the transaction
-    // so pick the first utxo as the from address
-    const utxo = baseTx.getOuts()[0];
-
-    if (!utxo.getAssetID().equals(this._transaction._assetId)) {
-      throw new Error('AssetID are not equals');
-    }
-    const secpOut = utxo.getOutput();
-    this._transaction._locktime = secpOut.getLocktime();
-    this._transaction._threshold = secpOut.getThreshold();
-    this._transaction._fromAddresses = secpOut.getAddresses();
-    this._transaction._rewardAddresses = (baseTx as AddDelegatorTx).getRewardOwners().getOutput().getAddresses();
-    this._transaction.setTransaction(tx);
```

Comment:
you removed these in transactionBuilder but used it in the specific builders.
However some are not set.

Reply:
The output is the changeoutput and could be empty. 
Now each  transaction type builder handle it


https://github.com/BitGo/BitGoJS/blob/7df888eca68bbc60505d837cdf8729c69f4d387e/modules/sdk-coin-avaxp/src/lib/exportTxBuilder.ts#L44

https://github.com/BitGo/BitGoJS/blob/7df888eca68bbc60505d837cdf8729c69f4d387e/modules/sdk-coin-avaxp/src/lib/delegatorTxBuilder.ts#L160